docs/private_shard/architecture.md

```diff
@@ -12,31 +13,35 @@ Due to the fact that by default the network’s token doesn’t have any monetar
 
 ## Validators
 Private shard validators will own some percentage of the stake (tokens) in the shard defined in the genesis of the network. The tokens are used for consensus only, and do not have any real monetary value. This is the default behavior and it is possible to create networks with actual monetary value if needed in cases of CBDC and similar. To run a minimal private shard you could run only one node controlling the network, but we recommend to run a minimum of four validators, as the network will halt if ⅓ of the nodes go down. This gives minimal redundancy and we recommend running as more validators as possible.
-
 For example, if a private shard is run by 100 validators owned by 100 different companies, it can be configured so that every single one of them has 1 percent of the total stake distributed. In this case every company will produce 1% of the blocks and more than 66% of the companies will need to behave maliciously and collude to corrupt the shard and 33% + 1 to stale the network.
-
 In the future we plan to implement different validator staking mechanisms. Being an owner of an NFT could be an entry ticket to become an validator in the shard. The NFT can represent anything from ownership of a game to having gone through KYC.
 
 ## Consensus
 Calimero is using the NEAR consensus algorithm called Nightshade. It is modeled as a single blockchain, in which each block logically contains all the transactions for all the shards, and changes the whole state of all the shards. Physically, however, no participant downloads the full state or the full logical block. Instead, each participant of the network only maintains the state that corresponds to the shards that they validate transactions for, and the list of all the transactions in the block is split into physical chunks, one chunk per shard. You can find more information about how it works on: https://near.org/papers/nightshade/#nightshade.
 
 ## Proof of Authority Shard
+In a proof of authority (PoA) shard, the validation of transactions and blocks is performed by a set of pre-approved validators or nodes, rather than through a competitive mining process. Validators are identified by their reputation, identity, or stake in the network, and are responsible for verifying the authenticity and validity of transactions. PoA shards are often used in private or permissioned blockchain networks, where the validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms, but may be more centralized and vulnerable to collusion or attacks by the validators.
```

Comment:
In our case PoA and PoS have same transaction speed and energy consumption, so I would remove that part. 

PoA => all validators same voting power and number of blocks produces 
PoS => validators can have customizable stake inside the network.

Reply:
I would also remove this part; but may be more centralized and vulnerable to collusion or attacks by the validators.